definition/IRoom.ts

```diff
@@ -154,12 +157,24 @@ export interface IOmnichannelRoom extends Omit<IRoom, 'default' | 'featured' | '
 	waitingResponse: any;
 	responseBy: any;
 	priorityId: any;
-	livechatData: any;
+	livechatData: { [k: string]: any };
```

Comment:
```suggestion
	livechatData: { [k: string]: unknown };
```

This should not affect anything else 🤔